### ISSUE TITLE
How to train your model?

### ISSUE THREAD

**Issue by Rainyfish** [uzh-rpg/rpg_e2vid#7]
Hi, I want to now how to train the model? can you release your train code of the model?

**Comment by westbrookS0** [comment]
me too

**Comment by 978326187** [comment]
me too

**Comment by spk921** [comment]
Yes, it will be really helpful if we know how to train the model.

**Comment by zhangxiann** [comment]
me too

**Comment by Adeelyousaf** [comment]
same

**Comment by LarryDong** [comment]
Hi guys, please check this repo from Timo&Cedric. It has a trainable e2vid model, but I'm not sure if it is exact the same as e2vid.
[https://github.com/TimoStoff/event_cnn_minimal](https://github.com/TimoStoff/event_cnn_minimal)

**Comment by hhsupremehh627** [comment]
me too

**Comment by Jee-King** [comment]
> Hi guys, please check this repo from Timo&Cedric. It has a trainable e2vid model, but I'm not sure if it is exact the same as e2vid. https://github.com/TimoStoff/event_cnn_minimal

@LarryDong  hi, have you trained e2vid successfully by this code https://github.com/TimoStoff/event_cnn_minimal?

**Comment by liuyezhe** [comment]
This work's training process are very similar to the paper "Learning Blind Video Temporal Consistency", and the training code are accessible. Here are the website: https://github.com/phoenix104104/fast_blind_video_consistency
